Public/Src/Cache/ContentStore/Interfaces/Results/DeleteResult.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using BuildXL.Cache.ContentStore.Hashing;
+
+namespace BuildXL.Cache.ContentStore.Interfaces.Results
+{
+        /// <summary>
+        ///     Result of the Delete call.
+        /// </summary>
+        public class DeleteResult : BoolResult
+        {
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="DeleteResult"/> class.
+        /// </summary>
+        public DeleteResult(ContentHash contentHash, long evictedSize, long pinnedSize, long replicaCount)
+        {
+            ContentHash = contentHash;
+            EvictedSize = evictedSize;
+            PinnedSize = pinnedSize;
+            ReplicaCount = replicaCount;
+        }

```

Comment:
I believe ReplicaCount is a fully internal concept. Not sure we want to surface it to deleters, or if there is a reason we should.